sqlalchemy_redshift/dialect.py

```diff
@@ -192,8 +192,8 @@ class TIMESTAMPTZ(sa.dialects.postgresql.TIMESTAMP):
 
     __visit_name__ = 'TIMESTAMPTZ'
 
-    def __init__(self):
-        super(TIMESTAMPTZ, self).__init__(timezone=True)
+    def __init__(self, timezone=True, precision=None):
+        super(TIMESTAMPTZ, self).__init__(timezone=True, precision=precision)
```

Comment:
I think we could ignore it since it would always be `True` (i.e. timezone will always be present) for TIMESTAMPTZ

Reply:
It would be worth adding a code comment about it. I assume that we need to accept a `timezone` param in the constructor in order to satisfy some assumption about the interface.

Reply:
> I assume that we need to accept a timezone param in the constructor in order to satisfy some assumption about the interface.

yep, that's exactly it. I'll add a comment explaining the reason for this

Reply:
comment added in 6fae561f7d4407ed4c034a4126c1b05a3fde78ef